src/entities/Proposal/routes.ts

```diff
@@ -580,7 +580,7 @@ async function getGrants(): Promise<CategorizedGrants> {
 async function getGrantsByUser(req: Request): ReturnType<typeof getGrants> {
   const address = req.params.address
   const isCoauthoring = req.query.coauthor === 'true'
-  if (!isEthereumAddress(address)) {
+  if (!address || address.length === 0 || !isEthereumAddress(address)) {
```

Comment:
Is the `address.length === 0` necessary? Isn't that caught in the isEthereumAddress function?